src/hooks.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 import { useCallback, useContext, useEffect, useState } from 'react';
-import * as optimizely from '@optimizely/optimizely-sdk';
```

Comment:
I think this should have caused a lint error below (using the variable name optimizely when its already declared in outer scope)

@mjc1283 - Is the tsc compiler smart enough to figure out which usage is a type?

Reply:
+1 to linting against this. In our example code we always use `optimizelySdk` as the name of the import.